### DISCUSSION TITLE
posts/2024/03/life-song-3/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#42] [General]
# posts/2024/03/life-song-3/

https://guozheng.netlify.app/posts/2024/03/life-song-3/

**Comment by liechi** [comment]
没怎么用过网易的音乐服务，不过这个一起听感觉很有意思。两个陌生人为什么会一起听歌呢？是因为审美一致吗？还是种陪伴？

想起研究生阶段有个师妹爱唱歌，常常相约傍晚，她唱，我听。我现在依然常常听歌，只是很少听现实里真人唱的了。

**Reply by residualsun1** [reply]
> 两个陌生人为什么会一起听歌呢？是因为审美一致吗？还是种陪伴？

我想更多的是陪伴。网易云能抓住当下年轻人的一个需求痛点——“网抑云”的调侃称号就来自感性的评论互动社区，在众多动情文案的加持下，音乐能放大网友的情绪（孤独？渴望得到聆听？渴望得到陪伴？），因此不少网友晚上使用网易云时会说“开e（emo）”，这时候一起听的功能可能就满足了他们被放大了孤独情绪后需要陪伴的情感。

当然啦，其实可能也很直接。我觉得大家现在多少都有些孤独寂寞，都不擅长在线下与他人产生情感链接，这样一种巧妙的方式可便利了原子化生活里个体对情感的需求。

> 想起研究生阶段有个师妹爱唱歌，常常相约傍晚，她唱，我听。

很浪漫的场景阿阿！不过既然是“相约”，想必是相互之间都心照不宣，羡慕了இ௰இ。

**Reply by liechi** [reply]
在线的交流虽然不能替代线下真实的朋友，但在倾述上却有其独特的优势，比如互相不认识，在分享的时候顾虑反而会少些；互相之前有距离感，也有神秘感，也较容易出美感。

>很浪漫的场景阿阿！不过既然是“相约”，想必是相互之间都心照不宣，羡慕了இ௰இ。

你的联想虽然合理，但在这个例子里却恰好不对。当年只是单纯地唱和听，不做它想。

**Reply by residualsun1** [reply]
想起初中读过一篇文章，就叫《人生若只如初见》。

> 你的联想虽然合理，但在这个例子里却恰好不对。当年只是单纯地唱和听，不做它想。

不过想想也觉得很美好。

**Reply by liechi** [reply]
你回复前不久，我正在看《饮水词》。我每次看纳兰词集，都会翻到“人生若只如初见”那篇。以前不甚明白“何事秋风悲画扇”里的悲是具体是什么情感？后来恍然，夏天扇子是刚需，秋风一起，画扇就要被遗弃了。这里的悲，用通俗的话讲，就是从小甜甜到牛夫人的悲。虽然全词也是这个意思，但突然发现这个细节，又觉有所得。
